src/util/config.tsx

```diff
@@ -0,0 +1,108 @@
+  "{config:option}`instance-security:security.csm":
+    "/reference/instance_options/#instance-security:security.csm",
+  "{config:option}`instance-security:security.idmap.isolated":
+    "/reference/instance_options/#instance-security:security.idmap.isolated",
+  "{config:option}`instance-security:security.nesting":
+    "/reference/instance_options/#instance-security:security.nesting",
+  "{config:option}`instance-security:security.privileged":
+    "/reference/instance_options/#instance-security:security.privileged",
+  "{config:option}`instance-security:security.secureboot":
+    "/reference/instance_options/#instance-security:security.secureboot",
+  "{config:option}`project-restricted:restricted.devices.disk":
+    "/reference/projects/#project-restricted:restricted.devices.disk",
+  "{config:option}`project-restricted:restricted.devices.nic":
+    "/reference/projects/#project-restricted:restricted.devices.nic",
+  "{ref}`cluster-evacuate": "/howto/cluster_manage/#cluster-evacuate",
+  "{ref}`cluster-https-address":
+    "/howto/cluster_config_networks/#cluster-https-address",
+  "{ref}`clustering-instance-placement":
+    "/explanation/clustering/#clustering-instance-placement",
+  "{ref}`clustering-instance-placement-scriptlet":
+    "/explanation/clustering/#clustering-instance-placement-scriptlet",
+  "{ref}`dev-lxd": "/dev-lxd/#dev-lxd",
+  "{ref}`howto-storage-buckets":
+    "/howto/storage_buckets/#howto-storage-buckets",
+  "{ref}`instance-options-limits-cpu":
+    "/reference/instance_options/#instance-options-limits-cpu",
+  "{ref}`instance-options-limits-cpu-container":
+    "/reference/instance_options/#instance-options-limits-cpu-container",
+  "{ref}`instance-options-qemu":
+    "/reference/instance_options/#instance-options-qemu",
+  "{ref}`instance-options-snapshots-names":
+    "/reference/instance_options/#instance-options-snapshots-names",
+  "{ref}`instances-limit-units":
+    "/reference/instance_units/#instances-limit-units",
+  "{ref}`metrics": "/metrics/#metrics",
+  "{ref}`network-bgp": "/howto/network_bgp/#network-bgp",
+  "{ref}`network-dns-server": "/howto/network_zones/#network-dns-server",
+  "{ref}`server-expose": "/howto/server_expose/#server-expose",
+};
+
+export const configDescriptionToHtml = (input: string, docBaseLink: string) => {
+  // special characters
+  let result = input
+    .replaceAll("<", "&lt;")
+    .replaceAll(">", "&gt;")
+    .replaceAll("\n", "<br>");
+
+  // documentation links
+  Object.entries(docLinkReplacements).forEach(([key, value]) => {
+    const href = `${docBaseLink}${value}`;
+    const linkText = key
+      .substring(key.lastIndexOf("`") + 1)
+      .split(":")
+      .pop()
+      ?.replaceAll("-", " ");
+    result = result.replaceAll(
+      key + "`",
+      `<a href="${href}" target="_blank" rel="noreferrer">${linkText}</a>`,
+    );
+  });
+
+  // code blocks
+  let count = 0;
+  while (result.includes("`") && count++ < 100) {
```

Comment:
Is this `100` an arbitrary limit, or where does it come from?

Reply:
Yes, I put it in to avoid infinite loops. It's a tactic of defensive programming.

Reply:
Maybe put it as a `const` at the top with a comment explaining the choice, so that we don't have magic values scattered in the code?